index.css

```diff
@@ -0,0 +1,167 @@
+body {
+  width: 375px;
+}
```

Comment:
Kindly delete the margin around the body to prevent overflow like below. Let's add `margin: 0px`to body to delete the margin around.
![Screen Recording 2022-12-02 at 2 24 36 PM](https://user-images.githubusercontent.com/25887253/205284145-c1e97513-95ae-4d9f-a9c7-c54a57d8486f.gif)